Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+- The following methods are now unconditionally `const fn`:
```

Comment:
I think it would be more clear to also mention that the `nightly` feature is no longer required for these functions, e.g.:

```suggestion
- The following methods no longer require the `nightly` feature to be `const fn`s`:
```
Or:

```suggestion
- The following methods are now unconditionally `const fn` (even without the `nightly` feature):
```